tests/operators/test_plots.py

```diff
@@ -71,6 +71,41 @@ def test_postage_stamp_contour_plot_sequence_coord_check(cube, tmp_working_dir):
         plot.spatial_contour_plot(cube)
 
 
+def test_spatial_pcolormesh_plot(cube, tmp_working_dir):
+    """Plot spatial pcolormesh plot of instant air temp."""
+    # Remove realization coord to increase coverage, and as its not needed.
+    cube.remove_coord("realization")
+    cube_2d = cube.slices_over("time").next()
+    plot.spatial_pcolormesh_plot(cube_2d, filename="plot")
+    assert Path("plot_462147.0.png").is_file()
+
+
+def test_pcolormesh_plot_sequence(cube, tmp_working_dir):
+    """Plot sequence of pcolormesh plots."""
+    plot.spatial_pcolormesh_plot(cube, sequence_coordinate="time")
+    assert Path("untitled_462147.0.png").is_file()
+    assert Path("untitled_462148.0.png").is_file()
+    assert Path("untitled_462149.0.png").is_file()
+
+
+def test_postage_stamp_pcolormesh_plot(monkeypatch, tmp_path):
+    """Plot postage stamp plots of ensemble data."""
+    ensemble_cube = read.read_cube("tests/test_data/exeter_em*.nc")
+    # Get a single time step.
+    ensemble_cube_3d = next(ensemble_cube.slices_over("time"))
+    monkeypatch.chdir(tmp_path)
+    plot.spatial_pcolormesh_plot(ensemble_cube_3d)
+    assert Path("untitled_463858.0.png").is_file()
+
+
+def test_postage_stamp_pcolormesh_plot_sequence_coord_check(cube, tmp_working_dir):
+    """Check error when cube has no time coordinate."""
+    # What does this even physically mean? No data?
```

Comment:
Guess any cube should always have time as a coordinate, even if its a scalar, as all output has to have some kind of valid time. But good to test error in case cube is corrupt